SpatialGDK/Source/SpatialGDK/Public/Tests/SpatialView/TestConnectionHandler.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/ConnectionHandler/AbstractConnectionHandler.h"
+#include "SpatialView/OpList/EntityComponentOpList.h"
+#include "SpatialView/WorkerView.h"
+
+namespace SpatialGDK
+{
+using FSendMessageCallback = TUniqueFunction<void(TUniquePtr<MessagesToSend>)>;
+
+class FTestConnectionHandler : public AbstractConnectionHandler
+{
+public:
+	using FSendMessageCallback = TUniqueFunction<void(TUniquePtr<MessagesToSend>)>;
+
+	explicit FTestConnectionHandler(FSendMessageCallback InSendMessageCallback = FSendMessageCallback(), FString InWorkerId = {},
+									Worker_EntityId InWorkerSystemEntityId = 0)
+		: WorkerId(InWorkerId)
+		, WorkerEntityId(InWorkerSystemEntityId)
+		, SendMessageCallback(MoveTemp(InSendMessageCallback))
+	{
+	}
+
+	virtual void Advance() override {}
+
+	virtual uint32 GetOpListCount() override { return PendingOpLists.Num(); }
```

Comment:
func should be const?

Reply:
the function it's overriding isn't const iirc. We can make this one const though.